doc/getting-started.md

```diff
@@ -9,133 +9,142 @@ mike@indexdata.com
+Set up tenant and URL for the remote backend by editing `.okapi`:
 
-	curl -w '\n' -D - -X POST -d '{"username":"diku_admin","password":"********"}' -H "Content-type: application/json" -H "Accept: application/json" -H "x-okapi-tenant: diku" http://localhost:9130/authn/login
-	curl -w '\n' -H "X-Okapi-Tenant: diku" -H "X-Okapi-Token: XXXXXXXX" -H "Accept: */*" -H "Content-Type: application/json" http://localhost:9130/copycat/profiles
+    $ vi ~/.okapi
 
-## To unassociate, undeploy and remove a running module
+Example values:
 
-This may be necessary so that the module can be re-added in order to force new or changed permissions to be recognised. The process has several steps, since the module must first be unassociated from the tenant and then undeployed before it can be removed:
+    $ cat ~/.okapi
+    OKAPI_URL=https://folio-snapshot-okapi.dev.folio.org
+    OKAPI_TENANT=diku
 
-	curl http://localhost:9130/_/proxy/tenants/diku/modules | grep mod-copycat
-	# From the output, extract the module-ID for the next step
-	curl -X DELETE http://localhost:9130/_/proxy/tenants/diku/modules/mod-copycat-1.0.0-SNAPSHOT
+Login:
 
-	curl http://localhost:9130/_/discovery/modules | grep -i copycat
-	# From the output, find the deployment UUID for the next step
-	curl -X DELETE http://localhost:9130/_/discovery/modules/mod-copycat-1.0.0-SNAPSHOT/15272446-17ba-41b8-b78b-b67df95bbeef
-	# Or undeploy ALL running instances by omitting the UUID of this specific one:
-	curl -X DELETE http://localhost:9130/_/discovery/modules/mod-copycat-1.0.0-SNAPSHOT
+    $ okapi-curl login
 
-	curl -X DELETE http://localhost:9130/_/proxy/modules/mod-copycat-1.0.0-SNAPSHOT
+Login will store token also in `~/.okapi`.
 
-## Adding permissions on the UI side
+Obtain User ID for the user (that you used in login):
 
-This is most easily done using the UI module's `yarn install`ed the Stripes CLI. Updating the module descriptor is straightforward, but it is then necessary to disable and re-enable the module for a tenant before that tenant can see the new permission definitions. Frustratingly, at the time of writing a bug in `stripes` means that it does not presently cache the Okapi URL or tenant name, so these need to be repeatedly provided on the command line:
+    $ okapi-curl /users?query=username%3D%3Duser | jq .users[0].id
 
-	yarn stripes mod update --okapi http://localhost:9130
-	yarn stripes mod disable --okapi http://localhost:9130 --tenant diku
-	yarn stripes mod enable --okapi http://localhost:9130 --tenant diku
-	echo ui-inventory.settings.single-record-import | yarn stripes perm assign --user diku_admin --okapi http://localhost:9130 --tenant diku
+Finally, we're ready to make an import. Replace userid - no quotes. We
+import LC number `2004436018 `- title `Ole Lukøie`.
 
+	$ okapi-curl /copycat/imports -HX-Okapi-User-Id:userid \
```

Comment:
Why do we need to specify `X-Okapi-User-Id` — isn't that implied by the token?

Reply:
SRS wants this. But I just realized that mod-copycat could start figuring it out for us. Yet another thing that's slightly sub optimal.